### DISCUSSION TITLE
Hello, I encountered auditory hallucinations while using the large-v3 model

### DISCUSSION THREAD

**Discussion by luyinla2017** [openai/whisper#1779] [General]
In the large-v3 model, there is a serious problem of repeated recognition results and sentence inversion in automatic language judgment for Chinese. large-v2 does not have this problem. Thank you

**Comment by bizrockman** [comment]
This is also true in German. It not only repeats sentences but also sometimes hallucinates entire paragraphs.

**Reply by Arche151** [reply]
Exact same experience with German transcriptions here! Really disappointing, since large-v2 doesn't have the issue.
